init.go

```diff
@@ -169,10 +180,16 @@ func (a *initFnCmd) buildFuncFile(c *cli.Context) error {
 		return errors.New("function name cannot contain a colon")
 	}
 
+        //if Dockerfile presents, force runtime to be 'docker'
 	if exists("Dockerfile") {
-		fmt.Println("Dockerfile found. Let's use that to build...")
-		return nil
-	}
+		fmt.Println("Dockerfile found, will use that to build. Force runtime 'docker'")
```

Comment:
Probably doesn't need to be anything forceful, "Using runtime 'docker'" may be a bit less jarring.

Reply:
updated the message to say "Docker found. Using runtime 'docker'"